spec/features/items/index_spec.rb

```diff
@@ -3,60 +3,122 @@
-        expect(page).to have_link(@brian.name)
-        expect(page).to have_css("img[src*='#{@pull_toy.image}']")
+        expect(page).to have_content("Inventory: #{@item_6.inventory}")
+        expect(page).to have_link(@pet_shop.name)
+        expect(page).to have_css("img[src*='#{@item_6.image}']")
       end
 
-      expect(page).to_not have_link(@dog_bone.name)
-      expect(page).to_not have_content(@dog_bone.description)
-      expect(page).to_not have_content("Price: $#{@dog_bone.price}")
+      expect(page).to_not have_link(@item_11.name)
+      expect(page).to_not have_content(@item_11.description)
       expect(page).to_not have_content("Inactive")
-      expect(page).to_not have_content("Inventory: #{@dog_bone.inventory}")
-      expect(page).to_not have_css("img[src*='#{@dog_bone.image}']")
+      expect(page).to_not have_content("Inventory: #{@item_11.inventory}")
+      expect(page).to_not have_css("img[src*='#{@item_11.image}']")
     end
 
     it "can have image links to item show page" do
-      within "#item-#{@pull_toy.id}" do
-        expect(page).to have_css("img[src*='#{@pull_toy.image}']")
+      within "#item-#{@item_1.id}" do
+        expect(page).to have_css("img[src*='#{@item_1.image}']")
         click_link('item-image')
       end
-        expect(current_path).to eq("/items/#{@pull_toy.id}")
+        expect(current_path).to eq("/items/#{@item_1.id}")
+    end
+
+    it "can see an area with most popular items and their qty purchased" do
+      within ".most-popular" do
+        expect(page).to have_link(@item_1.name)
+        expect(page).to have_content("Amount Purchased: 510")
+        expect(page).to have_link(@item_10.name)
+        expect(page).to have_content("Amount Purchased: 100")
+        expect(page).to have_link(@item_9.name)
+        expect(page).to have_content("Amount Purchased: 90")
+        expect(page).to have_link(@item_8.name)
+        expect(page).to have_content("Amount Purchased: 80")
+        expect(page).to have_link(@item_7.name)
+        expect(page).to have_content("Amount Purchased: 70")
+
+      end
+    end
+      
+    it "can see a section with least popular items and their qty purchased" do 
+      within ".least-popular" do
+        expect(page).to have_link(@item_2.name)
+        expect(page).to have_content("Amount Purchased: 20")
+        expect(page).to have_link(@item_3.name)
+        expect(page).to have_content("Amount Purchased: 30")
+        expect(page).to have_link(@item_4.name)
+        expect(page).to have_content("Amount Purchased: 40")
+        expect(page).to have_link(@item_5.name)
+        expect(page).to have_content("Amount Purchased: 50")
+        expect(page).to have_link(@item_6.name)
+        expect(page).to have_content("Amount Purchased: 60")
+      end
     end
+
+    # add test that "other section" only includes items not in most or least popular section *IF* we want to interpret the US as this
```

Comment:
Is there also a way to sad path test this?

Reply:
Per our convo with Brian, we don't need to worry about excluding popular and unpopular items from the "other" section